src/views/ApartmentApp/components/StudentApplication/index.js

```diff
@@ -21,7 +21,17 @@ import Agreements from './components/Agreements';
 import SaveButton from './components/SaveButton';
 import housing from '../../../../services/housing';
 import user from '../../../../services/user';
+
 const MAX_NUM_APPLICANTS = 8;
+const BLANK_APPLICATION_DETAILS = {
+  ApplicationID: null,
+  DateSubmitted: null,
+  DateModified: null,
+  EditorUsername: null,
+  EditorEmail: null,
+  Applicants: [],
+  ApartmentChoices: [],
+};
```

Comment:
I love this solution, this is exactly the kind of redundant/repeated code that had been bothering me